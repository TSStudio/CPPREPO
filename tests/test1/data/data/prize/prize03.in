5
lion 90 80 Y
Mac 90 80 N
Mis 82 81 Y
Mos 82 81 N
linda 85 83 N