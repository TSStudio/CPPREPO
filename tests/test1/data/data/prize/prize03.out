Mis
2000
2000